spaceship.cpp

```diff
@@ -0,0 +1,51 @@
+#include<iostream>
+#include<windows.h>
+#include<conio.h>
+#include "spaceship.h"
+using namespace std;
+
+//DECLARING OF GOTO FUNCTION TO MOVE LOCATIONS
+void gotoxy(int x, int y)
+{
+    COORD c={x,y};
+    SetConsoleCursorPosition (GetStdHandle(STD_OUTPUT_HANDLE),c);
+}
+
+
```

Comment:
extra empty line

Reply:
Ok will remove them, those lines came when i copied code from codeblocks, because i had compiled it there first.